src/components/Modals/ReleaseDetailsModal/ReleaseDetailsModalUpgradeOptionsVersion.tsx

```diff
@@ -19,11 +19,12 @@ interface IReleaseDetailsModalUpgradeOptionsVersionProps
   extends React.ComponentPropsWithoutRef<'span'> {
   version: string;
   isBeta: boolean;
+  unauthorized?: boolean;
 }
 
 const ReleaseDetailsModalUpgradeOptionsVersion: React.FC<
   IReleaseDetailsModalUpgradeOptionsVersionProps
-> = ({ version, isBeta, ...rest }) => {
+> = ({ version, isBeta, unauthorized, ...rest }) => {
```

Comment:
Using the `unauthorized` prop/variable to signify lacking permissions. This is different from other places where we use the more specific `canGetResource`, but this makes it easier to handle UI variations based on permissions in components shared with the non-MAPI UI.